src/styles/catalog.scss

```diff
@@ -0,0 +1,156 @@
+.catalog__product,
+.catalog {
+  display: flex;
+}
+
+.catalog__model,
+.catalog__code,
+.catalog__count,
+.catalog__name,
+.price__value {
+  margin: 0;
+}
```

Comment:
you need to reset margins by tags, not by classes

Reply:
Not fixed.